remote-workstations/new-agent-vm/server2016-graphics-agent.json

```diff
@@ -80,37 +88,10 @@
                 "description": "The password for the administrator account of the new VM and the domain"
             }
         },
-        "userStorageAccountName": {
-            "type": "string",
-            "metadata": {
-                "description": "Name of user's storage account"
-            },
-            "defaultValue": ""
-        },
-        "userStorageAccountKey": {
-            "type": "securestring",
-            "metadata": {
-                "description": "Key for accessing user's storage account"
-            },
-            "defaultValue": ""
-        },
-        "userStorageAccountUri": {
-            "type": "string",
-            "metadata": {
-                "description": "URI of the user's storage account"
-            }
-        },
-        "userStorageAccountSasToken": {
-            "type": "securestring",
-            "metadata": {
-                "description": "SAS token for accessing user's storage account"
-            },
-            "defaultValue": "?public"
```

Comment:
?public wasn't needed?

Reply:
Apparently it was a workaround for an issue that's been fixed